Core/Src/nero.c

```diff
@@ -9,22 +9,37 @@
 #include "cerberus_conf.h"
 #include "monitor.h"
 #include "pedals.h"
+#include "string.h"
```

Comment:
Need this?

Reply:
This is needed for memcpy for the data in the can msg